libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/SpeechPlayerProvider.java

```diff
@@ -42,8 +45,20 @@ public SpeechPlayerProvider(@NonNull Context context, String language,
     initialize(context, language, voiceLanguageSupported, voiceInstructionLoader);
   }
 
+  // Package private (no modifier) for testing purposes
+  SpeechPlayerProvider(@NonNull Context context, String language,
+                       boolean voiceLanguageSupported, VoiceInstructionLoader voiceInstructionLoader,
+                       ConnectivityStatusProvider connectivityStatus) {
+    initialize(context, language, voiceLanguageSupported, voiceInstructionLoader);
```

Comment:
Could this be `this(context, language, voiceLanguageSupported, voiceInstructionLoader);` instead?